.ci/flake8_wrapper_docstrings.sh

```diff
@@ -7,7 +7,7 @@ set -e
 # D3XX - Quoting issues.
 # D401 - First line should be in imperative mood 
 # D403 - First word of the first line should be properly capitalized
-args="--ignore=D --select=D100,D201,D202,D206,D207,D208,D209,D211,D3,D401,D403"
+args="--ignore=D --select=D100,D201,D202,D206,D207,D208,D209,D211,D3,D403"
```

Comment:
I considered that - I think lib/galaxy/containers previously conformed to that though right - this passed recently. So I guess the update changed what is considered "imperative mood" - and so we should probably re-address those comments and add this back in in a separate PR if it is important?